modules/getting-started/proc-add-custom-app-file-openshift-operator.adoc

```diff
@@ -0,0 +1,33 @@
+[id='proc-add-custom-app-file-openshift-operator_{context}']
+= Adding a custom application configuration file to {ocp-short} using the Operator
+
+You can use the {product} Operator to add a custom application configuration file to your {ocp-short} instance.
+
+.Prerequisites
+
+* You have created an {ocp-brand-name} account.
+
+.Procedure
+
+. From the {ocp-short} web console, select the *ConfigMaps* tab.
+. Click *Create ConfigMap*.
+. From *Create ConfigMap* page, select the *YAML view* option in *Configure via* and make changes to the file, if needed.
+. Click *Create*.
+. Go to the Backstage Custom Resource (CR) instance on the {ocp-short} web console or enter the following command on the command line:
++
+----
+oc edit Backstage rhdh
```

Comment:
This is just an example, right? Because the CR name might not necessarily be `rhdh`. So maybe add a placeholder and explain that it needs to be replaced with the actual CR name?

Reply:
Ah, ok. So _rhdh_ should be replaced with a variable in this case? Like _<custom resource name>_?

Reply:
> Ah, ok. So _rhdh_ should be replaced with a variable in this case? Like _< custom resource name >_?

Yes, that's what I was thinking.